plugin/.eslintrc.js

```diff
@@ -1,7 +1,8 @@
 module.exports = {
   root: true,
   extends: '../.eslintrc.js',
-  rules: {
-    'curly': 'error',
+  rules:{
+    '@typescript-eslint/no-use-before-define':'off'
```

Comment:
In the documentation is information that you need to disable more than this to make it wokrs - https://typescript-eslint.io/rules/no-use-before-define/#how-to-use